src/main/java/com/amihaiemil/versioneye/RtRepository.java

```diff
@@ -116,4 +128,17 @@ public boolean fork() {
     public JsonObject json() {
         return this.repository;
     }
+
+    @Override
+    public void delete(final String branchName) throws IOException {
+        String repoKey = this.fullname().replace('.', '~').replace('/', ':');
+        this.req.uri()
+            .path(repoKey)
+            .queryParam(
+                "branch",
```

Comment:
@SherifWaly don't these 2 fit on the same line? "branch", branch.replace(...)?
And I think the methods should be ``replaceAll(...)``, not ``replace(...)``, no?